cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -30,7 +30,7 @@
     SSL_CLIENT_VERIFICATION,
     SSL_ENABLED,
 )
-from cfy_manager.exceptions import FileError
+from cfy_manager.exceptions import FileError, ProcessExecutionError
```

Comment:
maybe push it down a couple of lines, so it will still be ordered by len

Reply:
Length ordering is really not a sensible idea.

Reply:
Also, it's not ordered by length at the moment.